cypress/tsconfig.json

```diff
@@ -2,6 +2,7 @@
 	"compilerOptions": {
 		"resolveJsonModule": true,
 		"module": "commonjs",
+		"moduleResolution": "node",
```

Comment:
As of upgrading to Cypress v15 tsconfig.json `compilerOptions` started getting respected correctly https://github.com/cypress-io/cypress/pull/31590

This caused Webpack to fail as `moduleResolution` defaulted to `classic` as we did not set it.